### ISSUE TITLE
Could not connect to Content Server after upgrade

### ISSUE THREAD

**Issue by HuifangYeo** [atoti/atoti#15]
## Description
After upgrading to Atoti 0.3.1, I encountered the error "Could not connect to Content Server..." during cube.visualize().

This only happens when we persist the content server
```
config = create_config(metadata_db="./metadata.db")
session = atoti.create_session(config=config)
```
![image](https://user-images.githubusercontent.com/35210495/79290402-96803900-7efe-11ea-9de7-bcdc99f867a8.png)

## Context

- Atoti version: 3.0.1
- Python version: 3.7.6
- Operating System: Windows 10 Pro

## Logs (if relevant)
Refer to attached logs.
[ActiveUI_Report_4_15_2020__9_35_27_AM.zip](https://github.com/atoti/atoti/files/4478681/ActiveUI_Report_4_15_2020__9_35_27_AM.zip)

**Comment by fabiencelier** [comment]
Hello,
Thank you for reporting the issue !
Can you please provide the server logs, you can print them with `session.logs_tail(0)`

**Comment by HuifangYeo** [comment]
Hi Fabien, 

please see attached.
[cube_visualize_session.log](https://github.com/atoti/atoti/files/4480644/cube_visualize_session.log)

**Comment by fabiencelier** [comment]
To clarify, did you create the metadata DB before upgrading and now you can't reconnect or is it a new metadata DB ?
If it is a DB from an old version what is the edition (enterprise or community) of your old and new version ?

**Comment by HuifangYeo** [comment]
The metadata DB is created before the upgrade and now we cannot reconnect after upgrade.
Previously it was from the atoti-plus bundle, and my bad, after upgrade, I am now on CE bundle.
However, does that mean that if we prepared the notebook with enterprise version, users using community edition won't be able to connect to the db?

**Comment by fabiencelier** [comment]
Looks like a content server created in one edition is not compatible in the other edition because we either initialize it with role "Anonymous" for community or "ROLE_USER" for the enterprise edition, but community users don't have the ROLE_USER role and enterprise users don't have the Anonymous role.

Therefore in 0.3.1 it is not possible to do cross edition metadata DB.

I will investigate more to see what we want to do, but I think we need at least the upgrade from community to enterprise to work

**Comment by HuifangYeo** [comment]
Is there anyway for me to change the roles in the db? I would like to retain my bookmark.

**Comment by tibdex** [comment]
Until we have something better available, you can use the enterprise edition to connect to the metadata DB again, [export it's content](https://artifacts.activeviam.com/documentation/rest/5.9.0-RC2/content-server.html#files__path__get) and change the reader and owner roles in it to match the ones used in a metadata DB created by the community edition. Once you are done editing the JSON file, you can use the community edition again to create a new metadata DB file and [import the edited JSON file](https://artifacts.activeviam.com/documentation/rest/5.9.0-RC2/content-server.html#files__path____import__post).

**Comment by fabiencelier** [comment]
This is fixed in 0.4.0
